src/clib/pio.h

```diff
@@ -438,6 +438,12 @@ enum PIO_ERROR_HANDLERS
 #define PIO_FLOAT  NC_FLOAT
 #define PIO_REAL   NC_FLOAT
 #define PIO_DOUBLE NC_DOUBLE
+#define PIO_UBYTE  NC_UBYTE
```

Comment:
Not only that, but it seems the pnetcdf supports all the netcdf-4 types except string. Very nice.